roles/cobbler/tasks/main.yml

```diff
@@ -1,4 +1,6 @@
 ---
+- include: ipmi_secrets.yml
```

Comment:
True; I had put them there initially in case it needed to be included from a second place. I can move them though.